SchedulerJobs/SchedulerJobs.Services/ELinksService.cs

```diff
@@ -50,6 +60,23 @@ public async Task ImportJudiciaryPeopleAsync(DateTime fromDate)
                     
                     _logger.LogInformation("ImportJudiciaryPeople: Executing page {CurrentPage}", currentPage);
                     var peoples = await _peoplesClient.GetPeopleAsync(fromDate, currentPage);
+                    
+                    string fileName = $"page{currentPage}.json";
+                    if (_featureToggles.StorePeopleIngestion())
+                    {
+                        var clientResponse = await _peoplesClient.GetPeopleJsonAsync(fromDate, currentPage);
```

Comment:
Rather than calling this API twice (as it's already being called in line 62), it's probably better to replace line 62 with `await _peoplesClient.GetPeopleJsonAsync(fromDate, currentPage);` and then deserialise it into a peoples object.

Line 67 here can then reference the raw response from line 62.

Reply:
this will involve a lot of refactoring on the test side. Would you like I proceed?

Reply:
refactor code done